backend/minesearch/api/routes/consolidated_results.py

```
@@ -22,19 +22,82 @@
 logger = logging.getLogger(__name__)
 router = APIRouter()
 
+# CSV imports for Phase 16.3
+import csv
```

Ruff (0.12.2)

`csv` imported but unused

Remove unused import: `csv`

(F401)